lib/structs/icinga/downtime_details.py

```diff
@@ -0,0 +1,12 @@
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass
+class DowntimeDetails:
+    object_type: str
+    object_name: str
+    start_time: int
```

Comment:
Use a more specific type rather than a primitive type if at all possible, in this case keep the start time as a `Date` instead of an int

Reply:
start_time is a timestamp which is an int?

Reply:
@DavidFair is asking you to use a Date obect instead of unix timestamp which you're currently using

Reply:
Icinga is expecting a timestamp which is why its there, I convert to the timestamp in the workflow but I can move the conversion to the icinga_api schedule_downtime function if makes more sense there

Reply:
IMO yes, because it's an Icinga API implementation detail 
It also makes debugging easier because the debugger gives you the date instead of `1728051804` for example

Reply:
Think we should re-visit this later if its a problem